src/resources/parser/glb-container-resource.js

```diff
@@ -190,6 +191,22 @@ class GlbContainerResource {
         return GlbContainerResource.createSceneHierarchy(sceneClones, 'Entity');
     }
 
+    // get material variants
+    getMaterialVariants() {
+        return this.data.variants;
+    }
+
+    // apply material variant to mesh instances
+    applyMaterialVariant(name, instances) {
```

Comment:
if this is a public API, it needs a JSDocs documentation

Reply:
Done